src/IngredientEntity.php

```diff
@@ -0,0 +1,30 @@
+<?php
+
+namespace GRUB; 
+
+/* Class for IngredientEntities */
```

Comment:
Update this class docblock to match the format used in the hydrator

Reply:
Done